### PR TITLE
Remove partner pod dead code and doc update

### PR DESCRIPTION
**Pull request by edcdavid**
This is to remove the unused partner/orchestrator code in the tnf suite. The partner repo is still required to instantiate the debug pods.
The documentation is also updated to reflect the new mechanisms

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -40,12 +40,12 @@ RUN wget --directory-prefix=${TEMP_DIR} ${OC_DL_URL} && \
 ENV PATH=${PATH}:"/usr/local/go/bin":${GOPATH}/"bin"
 
 # Git identifier to checkout
-ARG TNF_VERSION
-ARG TNF_SRC_URL=https://github.com/test-network-function/test-network-function
+ARG TNF_VERSION=remove-partner-pod
```
Comment (javierpena) [inline Dockerfile:43]: I guess these changes will be reverted before merging, right? ;)

File: README.md [hunk 2]
```diff
@@ -295,7 +290,7 @@ control over the outputs, see the output of `test-network-function.test --help`.
 cd test-network-function && ./test-network-function.test --help
 ```
 
-*Gotcha:* The generic test suite requires that the CNF has both `ping` and `ip` binaries installed.  Please add them
+*Gotcha:* The generic test suite requires that the CNF has the `ip` binariy installed.  Please add them
```
Comment (javierpena) [inline README.md:293]: typo: "binariy" vs "binary"

File: Dockerfile (hunk 1 above)
Comment (hamadise) [reply Dockerfile:43]: right, this is used to allow .github CI to pass without issues.

File: README.md (hunk 2 above)
Comment (hamadise) [reply README.md]: fixed.

File: Dockerfile [hunk 3]
```diff
@@ -45,7 +45,7 @@ ARG TNF_SRC_URL=https://github.com/test-network-function/test-network-function
 ARG GIT_CHECKOUT_TARGET=$TNF_VERSION
 
 # Git identifier to checkout for partner
-ARG TNF_PARTNER_VERSION
+ARG TNF_PARTNER_VERSION=main
```
Comment (jun-rh) [inline Dockerfile:48]: this is not needed?

File: README.md [hunk 4]
```diff
@@ -295,7 +290,7 @@ control over the outputs, see the output of `test-network-function.test --help`.
 cd test-network-function && ./test-network-function.test --help
 ```
 
-*Gotcha:* The generic test suite requires that the CNF has both `ping` and `ip` binaries installed.  Please add them
+*Gotcha:* The generic test suite requires that the CNF has the `ip` binary installed.  Please add them
```
Comment (jun-rh) [inline README.md:293]: This paragraph should be removed now

File: README.md [hunk 5]
```diff
@@ -527,9 +522,9 @@ For example:
 TNF_DEFAULT_BUFFER_SIZE=32768 ./run-cnf-suites.sh -f diagnostic
 ```
 
-## Issue-161 Some containers under test do not contain `ping` or `ip` binary utilities
+## Issue-161 Some containers under test do not contain `ip` binary
```
Comment (jun-rh) [inline README.md:525]: And this

Comment (hamadise) [reply README.md:525]: fixed

File: README.md (hunk 4 above)
Comment (hamadise) [reply README.md:293]: fixed

File: Dockerfile (hunk 3 above)
Comment (hamadise) [reply Dockerfile:48]: before this change, we did not specify the branch, I guess it's defined as parameter when building the image, we can revert this change as well

Comment (hamadise) [reply Dockerfile:48]: what do you think ?

Comment (hamadise) [reply Dockerfile:48]: anyway, I'll wait for CI to pass. then remove it again and see the results.

**Review by hamadise — APPROVED** [review]

**Review by jun-rh — APPROVED** [review]

**Pull request merged**